### ISSUE TITLE
 Language usage in the use cases

### ISSUE THREAD

**Issue by iherman** [w3c/did-use-cases#27]
I must admit I had some difficulties reading the Use Cases themselves (i.e., section 7). Part of it is simply the language and terminology used: the text uses a very dense language, with references, terms, etc, that are not necessarily clear outside of the US. It makes it difficult to follow for non-native English speakers. (I know it is very difficult to pinpoint the issues precisely...)

Using such very English (or American? I wouldn't know) terms and situations like "sticky wicket", "downstream customer", "estate planning", "probate authority" should definitely be avoided or, if necessary, explained. An example of US-orientedness is 7.3: the very notion of separate "planned parenthood clinics" is not a general term in, e.g., Europe or Asia, and the name does not make it clear that these clinics may also treat issues like UTI (which is not necessarily related to "parenthood"). Let alone the fact that I, for example, have never heard of the fact that you can make an online diagnose of anything leading to prescription (I do not think something like that would be possible where I live).

Also, these use cases are not specifications: they do not have to be perfectly precise. Using terms like "cryptographically authenticate", "such as firmware hashes and checksums", "self-management of identifiers and cryptographic key material,", "pairwise-pseudonymous", "rotating keys" are mixing use cases and requirements with the solution, and that should not be the goal of the document. It also makes it very difficult to read, mainly for someone who wants to understand what requirements led to DIDs and does not arrive with the full knowledge of crypto and all the other niceties...

**Comment by jandrieu** [comment]
Good feeback. We'll work on a revision to make the language more accessible.

**Comment by philarcher** [comment]
I've made some changes in the direction you're suggesting (see https://philarcher.github.io/did-use-cases/). Sticky wicket has gone and I've removed some of the more "this is written by Americans for Americans" text - although some survives. There's no mention of planned parenthood, the state of Washington etc. But there's a load of stuff around fiduciary probate whatnot that I sure don't understand - I, at least, would need to help translating that into English.
